src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -237,6 +237,12 @@ public static void main(String[] args) throws Exception {
                 // Nothing to do here !
             }
 
+            try {
+              wrkld.setDeferConstraintChecks(xmlConfig.getBoolean("deferConstraintChecks"));
```

Comment:
Deferred constraints have a different meaning in postgres / YSQL. So, I prefer to use some other term here. Maybe "addFkAfterLoad"